pkg/install/quota_test.go

```diff
@@ -0,0 +1,292 @@
+package install
```

Comment:
this file seems a bit heavyweight for the functionality added, not sure if it can be made a little more maintainable.